### PR TITLE
Add command line options to exception and display commands

### PR DESCRIPTION
**Pull request by DanielBallaSZTE**
You can now set --display and --exception parameters in the command line.

JerryScript-DCO-1.0-Signed-off-by: Daniel Balla dballa@inf.u-szeged.hu

### CODE REVIEW

File: jerry-debugger/jerry-client-ws.py [hunk 1]
```diff
@@ -81,6 +81,10 @@ def arguments_parse():
                         help="disable stop when newline is pressed (default: %(default)s)")
     parser.add_argument("--color", action="store_true", default=False,
                         help="enable color highlighting on source commands (default: %(default)s)")
+    parser.add_argument("--display", action="store", default="0",
+                        help="set display range (default: %(default)s)")
+    parser.add_argument("--exception", action="store", default="0",
```
Comment (zherczeg) [inline jerry-debugger/jerry-client-ws.py]: Isn't 1 is the default for exception?

File: jerry-debugger/jerry-client-ws.py [hunk 2]
```diff
@@ -81,6 +81,10 @@ def arguments_parse():
                         help="disable stop when newline is pressed (default: %(default)s)")
     parser.add_argument("--color", action="store_true", default=False,
                         help="enable color highlighting on source commands (default: %(default)s)")
+    parser.add_argument("--display", action="store", default="0",
```
Comment (zherczeg) [inline jerry-debugger/jerry-client-ws.py]: I think display should have an empty default.

File: jerry-debugger/jerry-client-ws.py [hunk 3]
```diff
@@ -81,6 +81,10 @@ def arguments_parse():
                         help="disable stop when newline is pressed (default: %(default)s)")
     parser.add_argument("--color", action="store_true", default=False,
                         help="enable color highlighting on source commands (default: %(default)s)")
+    parser.add_argument("--display", action="store", default="",
```
Comment (galpeter) [inline jerry-debugger/jerry-client-ws.py]: The default should be `None`. Also you can specify the type for which the value will be converted automatically (just add the `type=int` as a keyword argument after the `default` entry.

File: jerry-debugger/jerry-client-ws.py [hunk 4]
```diff
@@ -81,6 +81,10 @@ def arguments_parse():
...
+                        help="set display range")
+    parser.add_argument("--exception", action="store", default="",
```
Comment (galpeter) [inline jerry-debugger/jerry-client-ws.py]: As I see the `--exception` can only have two value: `0` and `1`. Could you please update that only there values are allowed (there is a choice action) also the default should be None and we can specify the type to be int. If the choices are present then we would not need for the try-except below and the help can be extended to contain the meaning of the values.

Comment (DanielBallaSZTE) [inline jerry-debugger/jerry-client-ws.py]: Thanks, added it to the code.

File: jerry-debugger/jerry-client-ws.py [hunk 5]
```diff
@@ -81,6 +81,10 @@ def arguments_parse():
                         help="disable stop when newline is pressed (default: %(default)s)")
     parser.add_argument("--color", action="store_true", default=False,
                         help="enable color highlighting on source commands (default: %(default)s)")
+    parser.add_argument("--display", action="store", default=None, type=int,
+                        help="set display range")
+    parser.add_argument("--exception", action="store", default=None, type=int, choices=range(0, 2),
+                        help="set exception config, usage 1: [Enable] or 0: [Disable]:")
```
Comment (galpeter) [inline jerry-debugger/jerry-client-ws.py]: Two minor things:
* No need for range call, just simply: `[0, 1]`
* There is an extra `:` at the end of the help message.

**Comment by DanielBallaSZTE** [comment]
Fixed those issues.

**Pull request merged**